packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header-item.component.tsx

```diff
@@ -16,6 +16,15 @@ const VitalsHeaderItem: React.FC<VitalsHeaderItemProps> = ({ interpretation, val
   const flaggedCritical = interpretation && interpretation.includes('critically');
   const flaggedAbnormal = interpretation && interpretation !== 'normal';
 
+  const generatedId = useId();
+
+  const labelId = `omrs-patient-chart-label-${unitName}-${generatedId}`;
+  const valueId = `omrs-patient-chart-value-${unitName}-${generatedId}`;
+  const unitId = `omrs-patient-chart-unit-${unitName}-${generatedId}`;
+
+  const displayValue =
+    value === null || value === undefined || value === '' ? t('notAvailable', 'Not available') : value;
```

Comment:
```suggestion
    Boolean(value) ? t('notAvailable', 'Not available') : value;
```

`Boolean` returns `false` for empty strings.